### ISSUE TITLE
Reproduction of results of Office31 and OfficeHome

### ISSUE THREAD

**Issue by andreamaracani** [roysubhankar/uncertainty-sfda#2]
Good morning and thank you for this work and for the release of the code.
 
I am currently trying to reproduce the results on the mentioned datasets using the provided code but I get accuracies that are way lower than the ones reported. I think I am doing something wrong or maybe I need to change some hyperparameters...
Do you have any suggestion I can try?

P.S.  The function `get_hessian` gave me error as it is and I had to rewrite it calling the `with` statement of `backpack` only on the backward of the loss. I do not know if this change can affect the results but I couldn't run the code otherwise...

My version is more or less:
```
    for i, (features, labels) in enumerate(feature_loader):
        
        classifier.zero_grad()
        logits = classifier(features)
        loss = criterion(logits, labels)

        with backpack(KFAC()):
            loss.backward()

        # compute and update U/V
        with torch.no_grad():
            # Hessian of classifier weights
            U_, V_ = W.kfac
            rho = min(1-1/(i+1), 0.95)

            U = rho * U + (1 - rho) * U_
            V = rho * V + (1 - rho) * V_
```

Thank you in advance for any help!

**Comment by andreamaracani** [comment]
I report the results that I am getting for Office-Home:

|         | AC   | AP   | AR   | CA   | CP   | CR   | PA   | PC   | PR   | RA   | RC   | RP   | AVG  |   
|---------|------|------|------|------|------|------|------|------|------|------|------|------|------|
| MY CODE | 52.7 | 74.8 | 77.4 | 63.5 | 70.6 | 73.4 | 59.7 | 49.8 | 75.2 | 66.0 | 54.5 | 74.6 | 66.0 |  
| PAPER   | 58.5 | 78.6 | 81.1 | 66.6 | 75.2 | 77.9 | 66.3 | 57.9 | 80.6 | 73.6 | 61.4 | 84.1 | 71.8 |
